torch/_torch_docs.py

```diff
@@ -3385,6 +3385,55 @@ def merge_dicts(*dicts):
     array([-1,  2,  3])
 """)
 
+add_docstr(torch.from_buffer,
```

Comment:
I just realised that the numpy function name doesn't have the `_`, could you update that to math the numpy naming? (https://numpy.org/doc/stable/reference/generated/numpy.frombuffer.html)

Reply:
Indeed, it doesn't have the `_`. I left it there, to keep consistency with `from_numpy`, though. What do you think?

Reply:
@mruberry what do you think?